lonely-lemmings/earlyinternet/gifapp/editorviews.py

```diff
@@ -0,0 +1,152 @@
+from base64 import b64decode, b64encode
+from io import BytesIO
+import json
+import os
+from typing import Union
+
+from PIL import Image as PILImage
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponse, HttpResponseNotFound, HttpResponseRedirect
+from django.shortcuts import render, redirect
+from django.views.decorators.csrf import csrf_exempt
+
+from .forms import ProjectForm
+from .models import Project, Image
+
+MEDIA_DIR = settings.MEDIA_ROOT
+
+
+@login_required
+def paint(request, project_name=None) -> HttpResponse:
+    context = {
+        "name": project_name
+    }
+    return render(request, "base_draw.html", context)
+
+
+@login_required
+@csrf_exempt
+def parse_save_request(request, project_name=None) -> HttpResponse:
+    """receives POST request of images as bytes, decodes and saves image paths to database"""
+
+    data = json.loads(request.body)
+    images_blob = data['image_BLOB']
+
+    # get project id
+    project = Project.objects.filter(name=project_name, user_id=request.user)[0]
```

Comment:
you should not inline comment things that is obvious. 
If its unclear that `project` is a project_id, then remove the commend and use the name `project_id`